test/E2ETest/TyeInitTests.cs

```diff
@@ -15,64 +18,32 @@ public class TyeInitTests
     {
         private readonly ITestOutputHelper output;
         private readonly TestOutputLogEventSink sink;
+        private readonly IDeserializer _deserializer;
 
         public TyeInitTests(ITestOutputHelper output)
         {
             this.output = output;
             sink = new TestOutputLogEventSink(output);
+            _deserializer = new DeserializerBuilder()
+                .WithNamingConvention(CamelCaseNamingConvention.Instance)
+                .Build();
         }
 
         [Fact]
-        public void SingleProjectInitTest()
+        public void Init_WorksForMultipleProjects()
         {
-            using var projectDirectory = CopySampleProjectDirectory(Path.Combine("single-project", "test-project"));
-
-            File.Delete(Path.Combine(projectDirectory.DirectoryPath, "tye.yaml"));
-
-            var projectFile = new FileInfo(Path.Combine(projectDirectory.DirectoryPath, "test-project.csproj"));
-
-            var (content, _) = InitHost.CreateTyeFileContent(projectFile, force: false);
-            var expectedContent = File.ReadAllText("testassets/init/single-project.yaml");
-
-            output.WriteLine(content);
-
-            Assert.Equal(expectedContent.NormalizeNewLines(), content.NormalizeNewLines());
-        }
-
-        [Fact]
-        public void MultiProjectInitTest()
-        {
-            using var projectDirectory = CopySampleProjectDirectory("multi-project");
-
-            // delete already present yaml
+            using var projectDirectory = CopyTestProjectDirectory("multi-project");
             File.Delete(Path.Combine(projectDirectory.DirectoryPath, "tye.yaml"));
 
             var projectFile = new FileInfo(Path.Combine(projectDirectory.DirectoryPath, "multi-project.sln"));
 
             var (content, _) = InitHost.CreateTyeFileContent(projectFile, force: false);
-            var expectedContent = File.ReadAllText("testassets/init/multi-project.yaml");
-
-            output.WriteLine(content);
+            var actual = _deserializer.Deserialize<ConfigApplication>(content);
 
-            Assert.Equal(expectedContent.NormalizeNewLines(), content.NormalizeNewLines());
-        }
-
-        [Fact]
-        public void FrontendBackendTest()
-        {
-            using var projectDirectory = CopySampleProjectDirectory("frontend-backend");
-
-            // delete already present yaml
-            File.Delete(Path.Combine(projectDirectory.DirectoryPath, "tye.yaml"));
-
-            var projectFile = new FileInfo(Path.Combine(projectDirectory.DirectoryPath, "frontend-backend.sln"));
-
-            var (content, _) = InitHost.CreateTyeFileContent(projectFile, force: false);
-            var expectedContent = File.ReadAllText("testassets/init/frontend-backend.yaml");
-
-            output.WriteLine(content);
+            var expectedContent = File.ReadAllText("testassets/init/multi-project.yaml");
```

Comment:
I feel like we should keep **ONE** of these as a text-bases comparison. There is some stuff we care about in there that's non-semantic (comments, empty properties vs the properties not present).